imsv-docs-docusaurus/openapi/endpoints/login/models/create-challenge-response.yaml

```diff
@@ -0,0 +1,39 @@
+type: object
+required:
+  - purpose
+  - address
+  - network
+properties:
+  id:
+    description: The unique identifier of the challenge.
+    type: string
+    example: "GlSDzZHcxrSDeJbKu"
+  purpose:
+    description: The purpose of the challenge.
+    type: string
+    example: "claim-web3-address"
+  address:
+    description: The web3 address to which the challenge is directed.
+    type: string
+    example: "0x123..."
+  network:
+    description: The blockchain network associated with the web3 address.
```

Comment:
What are the valid values?

Reply:
we should have an enum of the available networks for this @NicBathgate

Reply:
At the moment:
      - polygon-mainnet
      - polygon-mumbai
I've pushed a change to enum those two possible values, we will need to make sure we keep this up to date when we add new chains though (including when we replace mumbai with amoy soon)